README.md

```diff
@@ -1,2 +1,33 @@
-# portfolio-microverse
-Module 1: Portfolio
+![](https://img.shields.io/badge/Microverse-blueviolet)
+
+# Portfolio
+
+> "Portfolio" is the first big project in Microverse. Here I can show my overall HTML, CSS, Git, and GitHub knowledge for now.
+
+![screenshot](./portfolio-screenshot.jpg)
+
+## Live Demo
+
+[Click here to see a live demo](https://raw.githack.com/rdnrn/portfolio-microverse/mobile-version-skeleton/index.html)
+
+## Built With
+
```

Comment:
- [ ] As requested by the previous code reviewer which is a requirement for this project, please add steps on how to clone and run this project locally(i.e on the users machine)